src/helpers/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ref } from 'vue'
+export const MAX_POKEMON = 1302
```

Comment:
Since we are always getting a paginated list of Pokémon on page load that has "count" in the response, you could set this variable from the response instead of hardcoding incase it changes